package.json

```diff
@@ -24,11 +24,15 @@
   "author": "Felix Kling",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "@babel/parser": "^7.0.0",
-    "babel-plugin-transform-flow-strip-types": "^6.8.0",
-    "babel-preset-es2015": "^6.9.0",
-    "babel-preset-stage-1": "^6.5.0",
-    "babel-register": "^6.9.0",
+    "@babel/core": "^7.1.6",
+    "@babel/parser": "^7.1.6",
+    "@babel/plugin-proposal-class-properties": "^7.1.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/preset-env": "^7.1.6",
+    "@babel/preset-flow": "^7.0.0",
+    "@babel/preset-typescript": "^7.1.0",
+    "@babel/register": "^7.0.0",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
Is this needed or can I just remove it? @brieb

Reply:
it's needed if as a dep (it just links to `@babel/core`) for any dependency that uses `babel-core` as a dep like jest and hasn't explicitly upgraded to `@babel/core` themselves (https://github.com/facebook/jest/tree/master/packages/babel-jest#usage)

Reply:
Ok, thank you @hzoo!